behave_django/__init__.py

```diff
@@ -5,4 +5,4 @@
 __email__ = 'mixxorz@gmail.com'
 __url__ = 'https://github.com/behave/behave-django'
 __license__ = 'MIT'
-__version__ = '1.2.0.dev0'
+__version__ = '1.2.0.dev1'
```

Comment:
Please, make it `1.2.0` directly. I will rebase your commits onto master, so this will be the latest commit from the next released version.